### ISSUE TITLE
Axon node failed with "Integer overflow when casting to usize" after running for 3 months

### ISSUE THREAD

**Issue by serejke** [axonweb3/axon#1391]
### Current Behavior

```
[2023-08-25T12:19:26.847186817+00:00 ERROR panic] thread 'tokio-runtime-worker' panicked at 'Integer overflow when casting to usize': /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/primitive-types-0.12.1/src/lib.rs:38    0: core_run::Axon::panic_log
   1: core_run::Axon::set_ctrl_c_handle::{{closure}}::{{closure}}
   2: <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/alloc/src/boxed.rs:2001:9
      std::panicking::rust_panic_with_hook
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/panicking.rs:696:13
   3: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/panicking.rs:581:13
   4: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/sys_common/backtrace.rs:150:18
   5: rust_begin_unwind
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/panicking.rs:579:5
   6: core::panicking::panic_fmt
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/panicking.rs:64:14
   7: core_executor::utils::decode_revert_msg
   8: core_api::jsonrpc::error::vm_err
   9: core_api::jsonrpc::error::<impl core::convert::From<core_api::jsonrpc::error::RpcError> for jsonrpsee_core::error::Error>::from
  10: <core::pin::Pin<P> as core::future::future::Future>::poll
  11: <core_api::jsonrpc::impl::web3::Web3RpcImpl<Adapter> as core_api::jsonrpc::Web3RpcServer>::call::{{closure}}
  12: jsonrpsee_core::server::rpc_module::RpcModule<Context>::register_async_method::{{closure}}::{{closure}}
  13: jsonrpsee_server::transport::http::execute_call_with_tracing::{{closure}}::{{closure}}
  14: <futures_util::future::future::map::Map<Fut,F> as core::future::future::Future>::poll
  15: <futures_util::future::future::Map<Fut,F> as core::future::future::Future>::poll
  16: hyper::proto::h1::dispatch::Dispatcher<D,Bs,I,T>::poll_catch
  17: <hyper::server::conn::upgrades::UpgradeableConnection<I,S,E> as core::future::future::Future>::poll
  18: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  19: <tracing::instrument::Instrumented<T> as core::future::future::Future>::poll
  20: jsonrpsee_server::future::FutureDriver<F>::drive
  21: jsonrpsee_server::server::Server<B,L>::start_inner::{{closure}}
  22: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
  23: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
  24: tokio::runtime::task::harness::Harness<T,S>::poll
  25: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
  26: tokio::runtime::scheduler::multi_thread::worker::Context::run
  27: tokio::macros::scoped_tls::ScopedKey<T>::set
  28: tokio::runtime::scheduler::multi_thread::worker::run
  29: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
  30: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
  31: tokio::runtime::task::harness::Harness<T,S>::poll
  32: tokio::runtime::blocking::pool::Inner::run
  33: std::sys_common::backtrace::__rust_begin_short_backtrace
  34: core::ops::function::FnOnce::call_once{{vtable.shim}}
  35: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/alloc/src/boxed.rs:1987:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/alloc/src/boxed.rs:1987:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/sys/unix/thread.rs:108:17
  36: <unknown>
  37: <unknown>

[2023-08-25T12:19:26.847693062+00:00 INFO core_run] child thread panic, quit.
```

### Expected Behavior

Expected Behavior: never overflow

### OS

Linux

### Axon version

v0.1.0-beta.1

### Kernel

Linux

**Comment by KaoImin** [comment]
@serejke Panic seems to happen when call a rpc method. Is it right?

**Comment by Flouse** [comment]
### **Update**
The team is testing a hotfix branch for this problem. 
https://github.com/axonweb3/axon/commits/fix/v0.1.0-beta.1

Once it is ready, a docker image with a `v0.1.0-beta.1-hotfix` tag will be provided.

**Comment by Flouse** [comment]
> Once it is ready, a docker image with a `v0.1.0-beta.1-hotfix` tag will be provided.

ghcr.io/axonweb3/axon:0.1.0-beta.1-hotfix
https://github.com/axonweb3/axon/pkgs/container/axon/123972378?tag=0.1.0-beta.1-hotfix
This image is tested.
